pkg/processor/deployment/deployment.go

```diff
@@ -162,12 +163,21 @@ func (d deployment) Process(appMeta helmify.AppMetadata, obj *unstructured.Unstr
 		imagePullSecrets.ProcessSpecMap(specMap, &values)
 	}
 
+	if appMeta.Config().Probes {
+		err = probes.ProcessSpecMap(nameCamel, specMap, &values)
+		if err != nil {
+			return true, nil, err
+		}
+	}
+
 	spec, err := yamlformat.Marshal(specMap, 6)
 	if err != nil {
 		return true, nil, err
 	}
-	spec = strings.ReplaceAll(spec, "'", "")
 
+	spec = strings.ReplaceAll(spec, "'", "")
+	spec = strings.ReplaceAll(spec, "|\n        ", "")
+	spec = strings.ReplaceAll(spec, "|-\n        ", "")
```

Comment:
is it possible to manage tabs with helm? if this code relevant only for probe feature, please move it to probe processor

Reply:
no helm does not like slices with | so I had to remove them